pages/linux/pacman.md

```diff
@@ -14,9 +14,9 @@
 
 `pacman -Rs {{package_name}}`
 
-- Search the package database for a keyword:
+- Search the package database for a regular expression or keyword:
 
-`pacman -Ss icon theme`
+`pacman -Ss {{regexp}}`
```

Comment:
Quotation marks aren't needed (tested it right now), but wouldn't hurt either (being more future-proof?). Should we prioritize simplicity?

I kept the term "keyword" to handle simple examples, so "search_pattern" makes more sense, I agree :-)